js/WKWebView.ios.js

```diff
@@ -15,14 +15,20 @@ import { requireNativeComponent } from 'react-native';
 
 const RCTWKWebView = requireNativeComponent('RCTWKWebView');
 
-class WKWebView extends React.Component {
-  componentWillReceiveProps(nextProps) {
+type RCTWKWebViewProps = {
```

Comment:
Types, especially for Props should try to be read only and exact:

```
type RCTWKWebViewProps = $ReadOnly<{|
  allowsInlineMediaPlayback?: boolean,
  mediaPlaybackRequiresUserAction?: boolean,
  dataDetectorTypes?: boolean,
|}>;
```
This keeps people from accidentally passing in additional props or misspelling props. It also ensures that the component implementation doesn't do things like `this.props.foo = 4;`